js/scenes/LandingPage/LandingpageContainer.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import LandingPage from './LandingPage';
+
+class LandingPageContainer extends Component {
+
+
```

Comment:
More inconsistent whitespace.

Reply:
Fixed this specific instance of whitespace.